src/posts/paper/13-q-and-a.ts

```diff
@@ -179,6 +179,48 @@ However, we have retracted this method for the following reasons:
 * Various studies, such as [Baker et al.](https://www.acpjournals.org/doi/10.7326/M20-7145) have shown that, for at least some pockets of healthcare workers, healthcare workers are no more likely to get infected through their work than the average person is to get infected through socializing.
 * The above Washington State report shows that people who work outside the home got COVID at about the average rate for the state.
 
+# Vaccines Q&A
+
+## How should we think about people who have been vaccinated?
+
+**TL;DR**: Vaccinated people receive a multiplier on all incoming microCOVIDs from their activities. The multiplier depends on which vaccine they've gotten and is contingent on having waited long enough for their immune system to respond. Additionally, each microCOVID *may* be 2-4x less likely to land a vaccinated individual in a hospital.
+
+People who have been vaccinated are less likely to catch COVID and transmit it to others, which results in them getting fewer microCOVIDs from activities.
+However, they are not totally immune to COVID, and it is quite easy for a vaccinated person to counteract the decreased risk per activity by doing lots of
+risky activities (for instance, the Moderna vaccine confers a 1/5x multiplier for incoming microCOVIDs, so a vaccinated person who does 5x more risky behavior
+after being vaccinated would be just as many microCOVIDs as before).
+
+There are two questions that are important for understanding vaccines:
+
+1. How much less likely is it for a vaccinated individual to catch COVID? This determines the reduction in microCOVIDs a vaccinated individual receives from a given action.
+2. If a vaccinated individual contracts COVID, how much less (or more) likely is this to result in negative consequences (hospitalizaton or death). This determines how much a person can increase their microCOVID budget for being vaccinated (on top of being able to do more activities from part 1).
+
+### 1. How much less likely is it for vaccinated individuals to catch COVID? (Reduction in microCOVIDs)
+
+We do not know for sure how likely it is for a vaccinated individual to catch to COVID. Vaccinated individuals still have a chance of becoming sick (even if they are not seriously ill), and may be asymptomatic carriers of the virus, capable of infecting others without having symptoms themselves. None of the vaccine studies have directly addressed what the chances of transmission from a vaccinated individual are. We attempt to make educated guesses by pulling in other sources. For our derivation of these numbers, see the [Research Sources](/paper/14-research-sources#vaccines).
```

Comment:
"catch to COVID" → "catch COVID"

(I only caught it because I copy-pasted this whole thing into a spell-checker)